day2/q2.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+func main() {
+	var wg sync.WaitGroup
+	wg.Add(200)                   // add 200 students to wait group
+	rating := make(chan int, 200) // buffered channel for 200 students
```

Comment:
variable name as `rantings` will be more suitable here, as you using it to store multiple rantings

Reply:
okay understood. will change